MANIFEST.in

```diff
@@ -1,2 +1,5 @@
 recursive-include shoop *.jinja *.html *.txt
 recursive-include shoop_tests *.jinja *.html *.txt
+prune shoop/admin/static_src/*/bower_components
+prune shoop/admin/modules/*/bower_components
+prune shoop/*/node_modules
```

Comment:
How will those get installed then? I thought we need that stuff into the sdist package so that installing from sdist will have all the required resources.